docs/cloud-native-security/cspm-faq.asciidoc

```diff
@@ -21,6 +21,12 @@ After you deploy the CSPM integration, it can take up to 10 minutes for resource
 
 Newly unenrolled cloud accounts can take a maximum of 24 hours to disappear from the Cloud Security Posture dashboard.
 
+**Can I install an unlimited number of agentless integrations?**
+
+Each agentless integration deployment counts towards the maximum number of agentless integrations you can have in the same project. When you reach the maximum, you'll see an error message similar to `You have deployed the maximum number of agentless integrations. To continue, remove some or use agent-based deployment.` You won't be able to deploy another agentless integration until you remove an existing one. All <<agentless-integrations>> count towards this limit.
+
+
+
 
 [discrete]
 == KSPM FAQ
```

Comment:
I know this is outside of the scope of the PR, but did want to note [these](https://security-docs_bk_6712.docs-preview.app.elstc.co/guide/en/security/8.x/agentless-integrations.html#_beta_agentless_integrations) entries are formatted slightly differently because they're not preceded by an empty line.

Reply:
Good point, but I'm going to save this and your other comment for the future since this is a twin PR.

Reply:
works for me : )